src/components/buttons/critical-action-button/README.md

```diff
@@ -0,0 +1,49 @@
+# Buttons: Critical Action Button
+
+## Usage
+
+```js
+import { CriticalActionButton } from '@commercetools-frontend/ui-kit';
```

Comment:
Why do we have two different __usage__ sections?

Reply:
This is a copy and past of other components. I see many of them contains two section actually. We can fix them in a bulk in separate PR. Thanks for pointing this out

Reply:
that would be great :)